src/SourceDialog/SourceDialog.vala

```diff
@@ -285,7 +285,7 @@ public class Maya.View.SourceDialog : Gtk.Grid {
                 case "#ed5353":
                     color_button_strawberry.active = true;
                     break;
-                case "#f37329":
+                case "#ffa154":
```

Comment:
This adjustment only provides consistency. Unfortunately I cannot yet speak of WCAG compliance here. For this we would need a background of at least @ORANGE_700 with an assumed foreground color of #fff.